### PR TITLE
[Docs] Document Platform in the docs #3701

### PR DESCRIPTION
**Pull request by emilioicai**

### CODE REVIEW

**Comment by foghina** [comment]
@mkonicek LGTM, but don't we have this information somewhere already?

**Comment by satya164** [comment]
@foghina https://github.com/facebook/react-native/issues/3701

File: docs/PlatformSpecificInformation.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+---
+id: platform-specific-code
+title: Platform Specific Code
+layout: docs
+category: Guides
+permalink: docs/platform-specific-code.html
+next: native-modules-ios
+---
+
+When building a cross platform app, the need to write different code for different platforms may arise. This can always be achieved by organizing the various components in different folders:
```
Comment (ide) [inline docs/PlatformSpecificInformation.md]: cross platform -> cross-platform

File: docs/PlatformSpecificInformation.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+React Native will import the correct component for the running platform.
+
+## Platform module
+A module is provided by React Native to detect what is the platform in which the app is running. This piece of functionality can be useful when only small parts of a component are platform specific.
+
+```javascript
+var {Platform} = React;
+
+var styles = StyleSheet.create({
+    height: (Platform.OS === 'ios') ? 200 : 100
```
Comment (ide) [inline docs/PlatformSpecificInformation.md]: 2 spaces for indentation. also can you add a trailing comma at the end of this line?

File: docs/PlatformSpecificInformation.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+});
+```
+
+`Platform.OS` will be `ios` when running in iOS and `android` when running in an Android device or simulator.
+
+###Detecting android version
```
Comment (ide) [inline docs/PlatformSpecificInformation.md]: Capitalize "Android"

**Comment by ide** [comment]
Looking good. Eventually it would be great to have some docs for Platform under the "Modules" section of the docs as well but I can see this document being quite helpful already.

**Comment by ide** [comment]
@emilioicai have you had a change to update this? I'd like to merge your PR after you've made a few edits.

**Comment by emilioicai** [comment]
updated!

**Pull request merged**